packages/flutter/test/material/stepper_test.dart

```diff
@@ -972,6 +972,34 @@ testWidgets('Stepper custom indexed controls test', (WidgetTester tester) async
     }
   });
 
+  testWidgets('Vertical and Horizontal Stepper controller test', (WidgetTester tester) async {
+      ScrollController controller = ScrollController();
```

Comment:
Analyzer is failing here
```suggestion
      final ScrollController controller = ScrollController();
```

Reply:
Okay got it! thanks

Reply:
Added the change